src/RadixDlt.NetworkGateway.PostgresIntegration/Models/EntityResourceVaultAggregateHistory.cs

```diff
@@ -84,6 +89,107 @@ internal class EntityResourceVaultAggregateHistory
     [Column("resource_entity_id")]
     public long ResourceEntityId { get; set; }
 
+    /// <summary>
+    /// The aggregate list of all vaults ordered by most recently updated.
+    /// </summary>
     [Column("vault_entity_ids")]
     public List<long> VaultEntityIds { get; set; }
+
+    /// <summary>
+    /// The last significant update state version for resources stored in <see cref="VaultEntityIds"/>.
+    /// </summary>
+    /// <remarks>
+    /// This property is nullable in order to ensure smooth database migration process.
+    /// </remarks>
+    [Column("vault_significant_update_state_versions")]
+    public List<long>? VaultSignificantUpdateStateVersions { get; set; }
```

Comment:
Out of interest - what's the purpose of storing this?

* It's inaccurate - or at least "Significant" needs spelling out, because it's taken a while to work out that it means "the last time it was updated and overtook some other resource as the most recently updated"
* I can't see anywhere it's actually used, other than for a minor optimization which is ignored anyway because of the SequenceEqual

Am I missing something?